build.rs

```diff
@@ -52,68 +53,58 @@ fn build() {
     platform::print_libs(out_dir);
 }
 
+// Similar to unwrap, but panics on just the error value
+#[allow(dead_code)]
+fn err_to_panic<T, E: Display>(result: Result<T, E>) -> T {
+    match result {
+        Ok(x) => x,
+        Err(e) => panic!("{}", e)
+    }
+}
```

Comment:
`.unwrap()` has some extra information-- on `Err`, it calls `unwrap_failed("called `Result::unwrap()` on an `Err` value", e)`. I personally think that the plain old `Display` for `Err` reads much more nicely.